lib/main.js

```diff
@@ -108,10 +108,16 @@ module.exports = async (reporter, definition) => {
     function processType (typeName, typeDef) {
       for (const propName in typeDef) {
         const propDef = typeDef[propName]
-        if (propDef.type === 'Edm.String' && propDef.document) {
-          propDef.maxLength = 'max'
-        } else if (propDef.type === 'Edm.String' && typeName === 'SettingType' && propName === 'value') {
-          propDef.maxLength = 'max'
+        if (propDef.type === 'Edm.String') {
+          if (propDef.document) {
+            propDef.maxLength = 'max'
+          } else if (typeName === 'SettingType' && propName === 'value') {
+            propDef.maxLength = 'max'
+          } else if (typeName === 'ChangeType' && propName === 'serializedDoc') {
```

Comment:
I was checking this more deeply now, and the VersionType has a nested collection of complex types -> VersionType.changes. This works with SQL stores the way that it gets JSON.stringified into a single column.
This column should be clob in this case, because there is not `VersionType.changes_serializedDoc`.
The fixed looks like this for me.

```js
function processType (typeName, typeDef) {
      for (const propName in typeDef) {
        const propDef = typeDef[propName]
        if (propDef.type === 'Edm.String') {
          if (propDef.document) {
            propDef.maxLength = 'max'
          } else if (typeName === 'SettingType' && propName === 'value') {
            propDef.maxLength = 'max'
          }
        } else {
          if (typeName === 'VersionType' && propName === 'changes') {
            propDef.maxLength = 'max'
          }
        }
    }
```